locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
```

Comment:
Why are we hardcoding these variable values ... can we take a s an input from user ?

Reply:
Most of these are hardcoded for our container registry.  However, should a client need to perform the same install on an airgap or similar environment they will need to customize the repository?  The namespace can be pulled out in future updates, at the time this portion of the code was written that value was static.